docs/plugins/cli.md

```diff
@@ -0,0 +1,45 @@
+# CLI
+
+![CLI Screenshot](../assets/cli.gif)
+
+## Install
+The Devtools plugin can be installed using NPM:
```

Comment:
Should be "The CLI can be installed ..."

Reply:
done